static/searchPage.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <title>Ogene - Group Search</title>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDDhfcuk15Apx3i72mFSilulsPtJReGhcY"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=APIkey"></script>
+    <script src="../js/searchPage.js"></script>
```

Comment:
Think this line can be changed to `/js/searchPage.js`

Reply:
What was the resolution to this conversation (about putting the API key in the HTML file)?